lib/sharing/editor_doc_property.dart

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import '../editing/editor.dart';
+import '../elements/bind.dart';
+
+class EditorDocumentProperty implements Property {
```

Comment:
How extensively used are these little binder classes? Do we want to keep them around in the new playground? It's fine if we do -- I just want to point out that it's something we can also choose not to do.

Reply:
I would like to keep them - they are being used to bind the editor contents to the gist contents.

Reply:
👍